website/src/pages/Home/index.tsx

```diff
@@ -21,16 +21,22 @@ import { Workloads } from "./workloads";
 import { Metadata } from "../../components/Metadata";
 import SeoJson from "../../resources/seo.json";
 import { currentOrigin } from "../../utils/currentHost";
+import TestimonialSlider from '../../components/testimonialSlider';
+import { Testimonial } from "../../components//testimonialSlider/interface";
 
 const Home: React.FC = () => {
     const classes = useStyles();
     const { t } = useTranslation();
     const [tabValue, setTabValue] = useState<number>(0);
-    const [adopterTestimonials, setAdopterTestimonials] = useState<any>("");
     const [copyCommand, setCopyCommand] = useState({
         text: 'helm install stable/openebs --name openebs --namespace openebs',
         status: 'Copy to clipboard'
     });
+    const [adopterTestimonials, setAdopterTestimonials] = useState<Testimonial[]>([]);
+    useEffect(()=>{
+        setAdopterTestimonials(adopters);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why this is disabled??

Reply:
Instead, we can change the code accordingly instead of disabling it. 
cc @rakeshPRaghu

Reply:
PTAL added useCallback for setting the dependency check

Reply:
@AVRahul changing it back since the earlier way we are following in the project, and dont know if the useCallback is a solution.
cc: @IsAmrish

Reply:
Ok